libraries/vault_service.rb

```diff
@@ -58,7 +58,7 @@ class Chef::Resource::VaultService < Chef::Resource
   attribute(:source_url, kind_of: String)
 
   def command
-    "vault server -config=#{config_path}"
+    "/usr/local/bin/vault server -config=#{config_path}"
```

Comment:
This may be better if we do something along the lines of `/usr/bin/env vault` because then it should be configurable from the service environment.